internal/util/logging/handler.go

```diff
@@ -47,8 +47,9 @@ type Handler struct {
 //
 //nolint:vet // for readability
 type NewHandlerOpts struct {
-	Base         string // base handler to create: "console", "text", "json" or "mongo"
+	Base         string // base handler to create: "console", "text", "json", "mongo" or "embeddable"
 	Level        slog.Leveler
+	Handler      slog.Handler
```

Comment:
Let's be specific that `Handler` is only for embeddable base. We could name it `EmbeddedHandler`, and add a comment.